iidm/iidm-serde/src/test/resources/V1_14/manualFrequencyRestorationReserve.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_14" xmlns:tr="http://www.powsybl.org/schema/iidm/ext/manual_frequency_restoration_reserve/1_0" id="sim1" caseDate="2022-04-06T13:43:05.020+02:00" forecastDistance="0" sourceFormat="test" minimumValidationLevel="STEADY_STATE_HYPOTHESIS">
```

Comment:
Could you merge the main branch in your PR and move this file in the "V1_15" directory?
A new IIDM version will be introduced by the next PowSyBl release.